notebooks/huggingface_notebook.ipynb

```diff
@@ -0,0 +1,315 @@
+      "Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.weight', 'classifier.bias']\n",
+      "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import transformers\n",
+    "\n",
+    "# Create a BERT sequence classification model using HuggingFace transformers\n",
+    "model = transformers.AutoModelForSequenceClassification.from_pretrained('bert-base-uncased', num_labels=2)\n",
+    "tokenizer = transformers.AutoTokenizer.from_pretrained('bert-base-uncased') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use `HuggingFaceModel` to convert model to `ComposerModel`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torchmetrics import Accuracy\n",
+    "from torchmetrics.collections import MetricCollection\n",
+    "from composer.models.huggingface import HuggingFaceModel\n",
+    "from composer.metrics.nlp import LanguageCrossEntropy\n",
+    "\n",
+    "metrics = [LanguageCrossEntropy(vocab_size=tokenizer.vocab_size), Accuracy()]\n",
+    "# Package as a composer model\n",
+    "composer_model = HuggingFaceModel(model, metrics=metrics)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Optimizers and Learning Rate Schedulers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last setup step is to create an optimizer and a learning rate scheduler. We will use PyTorch's AdamW optimizer and linear learning rate scheduler since these are typically used to fine-tune BERT on tasks such as SST-2."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torch.optim import AdamW\n",
+    "from torch.optim.lr_scheduler import LinearLR\n",
+    "\n",
+    "optimizer = AdamW(\n",
+    "    params=composer_model.parameters(),\n",
+    "    lr=3e-5, betas=(0.9, 0.98),\n",
+    "    eps=1e-6, weight_decay=3e-6\n",
+    ")\n",
+    "linear_lr_decay = LinearLR(\n",
+    "    optimizer, start_factor=1.0,\n",
+    "    end_factor=0, total_iters=150\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Composer Trainer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now specify a Composer `Trainer` object and run our training! `Trainer` has many arguments that are described in our [documentation](https://docs.mosaicml.com/en/stable/api_reference/composer.trainer.trainer.html), but let's discuss the less obvious arguments used below:\n",
```

Comment:
Took me a second to understand this sentence, so this hopefully smooths it a bit
```suggestion
    "We will now specify a Composer `Trainer` object and run our training! `Trainer` has many arguments that are described in our [documentation](https://docs.mosaicml.com/en/stable/api_reference/composer.trainer.trainer.html), so we'll discuss only the less-obvious arguments used below:\n",
```